src/app/aircraft/edit/post.controller.js

```diff
@@ -12,24 +15,21 @@ module.exports = (req, res) => {
   const { craftReg } = req.body;
   const { craftType } = req.body;
   const craftBase = _.toUpper(req.body.craftBase);
-  const { craftId } = req.body;
```

Comment:
The craftId is an input that is hidden on the markup, which is populated via the cookie anyway, so is unnecessary.